docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -124,6 +124,55 @@ A closing curly bracket (`}`) is ignored if not inside an expression/tag.
 
 TIP: It is also possible to use escape sequences `\{` and `\}` to insert delimiters in the text. In fact, an escape sequence is usually only needed for the start delimiter, ie. `\\{foo}` will be rendered as `{foo}` (no evaluation will happen).
 
+=== Removing Standalone Lines From the Template
+
+By default, the parser removes standalone lines from the template output.
+A *standalone line* is a line that contains only section tags (e.g. `{#each}` and `{/each}`), parameter declarations (e.g. `{@org.acme.Foo foo}`) and whitespace characters.
+The default behavior can be disabled by setting the property `quarkus.qute.remove-standalone-lines` to `false`.
```

Comment:
I think adding a sentence mentioning, what disabling such a behaviour can result in, would be a good thing. I see that there's an example output a few lines down, which shows what happens, but explicitly stating something like  "If this property is disabled, then each standalone line will result in a new line character, in the generated output" might be helpful

Reply:
Yep, that makes sense. Except that it's not "a new line character" but "a bunch of whitespace chars resulting in a new line" ;-).